src/spawn.ts

```diff
@@ -160,18 +157,8 @@ export async function exportGenesisWasm(
 }
 
 /// Export the genesis state aka genesis head.
-export async function exportGenesisState(
-	bin: string,
-	id?: string,
-	chain?: string
-): Promise<string> {
+export async function exportGenesisState(bin: string): Promise<string> {
 	let args = ["export-genesis-state"];
-	if (id) {
-		args.push("--parachain-id=" + id);
-	}
-	if (chain) {
-		args.push("--chain=" + chain);
```

Comment:
Why do we need to remove chain selection from the parachain parameters? It will cause issues when you export the genesis state.

Reply:
According to a comment, it was't supported by cumulus anymore

Reply:
I think this was a misunderstanding. The only thing that got removed was the `parachain-id` https://github.com/paritytech/cumulus/pull/739
No need to remove the option to select a chain. It is very needed, in polkadot-collator we have several chains, the same stands for the polkadot binary.

Reply:
Indeed. How was this resolved, perhaps I'm missing something @joelamouche ?

Reply:
@ghzlatarev I left the --chain command. So if you specify the chain in the chain field of the parachain it should work

Reply:
@joelamouche I don't see it in `startCollator` or `exportGenesisWasm` , is it not needed there ?
Right now if I start a `westmint-local` network I get some default `Local Testnet`

Reply:
@joelamouche i hit this code path https://github.com/paritytech/cumulus/blob/master/polkadot-parachains/src/command.rs#L119 -> https://github.com/paritytech/cumulus/blob/master/polkadot-parachains/src/chain_spec.rs#L71

Reply:
You're right Im adding it back